Header header

uint32 gear
string status